### PR TITLE
Client edit

### PR DESCRIPTION
**Pull request by oliveiraigorm**
image

image

### CODE REVIEW

File: lib/Controller/PageController.php [hunk 1]
```diff
@@ -90,4 +90,36 @@ public function create(string $name, string $email, string $description): String
...
+		$client->setName($name);
+		$client->setDescription($description);
+		$client->setTimezone($timezone);
+		$client->setCountry($country);
+		$client->setCity($city);
+		$client->setAge($age);
+		$client->setContacts($contacts);
+		try {
+			$this->mapper->update($client);
+			return "Success";
```
Comment (frozenbanana) [inline lib/Controller/PageController.php:120]: Use the pattern found in the note tutorial app.
```
	public function update($id, $title, $content, $userId) {
		try {
			$note = $this->mapper->find($id, $userId);
			$note->setTitle($title);
			$note->setContent($content);
			return $this->mapper->update($note);
		} catch (Exception $e) {
			$this->handleException($e);
		}
	}
```

https://github.com/nextcloud/app-tutorial/blob/master/lib/Service/NoteService.php

**Review by frozenbanana — CHANGES_REQUESTED** [review]
Small change on controller method otherwise good.

File: lib/Controller/PageController.php (hunk 1 above)
Comment (oliveiraigorm) [reply lib/Controller/PageController.php]: done

**Review by frozenbanana — DISMISSED** [review]
👏

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**